src/Mod/TechDraw/App/DrawUtil.h

```diff
@@ -171,25 +174,11 @@ class TechDrawExport DrawUtil
                                       Base::Vector2d d2);
 
 
-    template <typename T>
-    static Base::Vector3d toVector3d(const T& v)
-    {
-        return Base::Vector3d(v.X(), v.Y(), v.Z());
-    }
-
     static Base::Vector3d toVector3d(const QPointF& v)
     {
         return Base::Vector3d(v.x(), v.y(), 0);
     }
 
```

Comment:
Is this still in use?

Reply:
Yep. If we want to remove this then we will need to specialize vec_traits for QPointF but specializing it like that to convert a 2d vector to vector3d seems kind of problematic. A better solution might be specializing Base::convertTo.
Tbh, I am not sure about this one. I will go with whatever you guys recommend.

Reply:
If it's in use we can keep it